README.md

```diff
@@ -68,12 +80,12 @@ your Github or Gitlab account. This will be known as our 'remote'.
 
 3. Add the remote (i.e. the new repo URL from your Github or Gitlab account) to your local repository. Run the following command:
 
-```git remote add origin <my github/gitlab URL>```
+```shellgit remote add origin <my github/gitlab URL>```
 
 When you push to your repo, note that the default branch is called 'main'.
 
 
-## Optional Configurations
+# Optional Configurations
```

Comment:
Remove this section, it's not making sense.

Reply:
Will do. Note that this section came from the template from the copier-agent. We will have to revisit that tool and adjust the template README accordingly. 